airflow/contrib/kubernetes/kubernetes_request_factory/kubernetes_request_factory.py

```diff
@@ -199,7 +201,7 @@ def extract_tolerations(pod, req):
             req['spec']['tolerations'] = pod.tolerations
 
     @staticmethod
-    def extract_security_context(pod, req):
+    def extract_security_context(pod: Pod, req):
```

Comment:
This is going to fail on Python2

Reply:
My bad. I forgot we still supported Python 2.

Reply:
Fixed.